apps/react-18-tests-v9/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-18-tests-v9",
-  "description": "React 18 test application",
-  "version": "1.0.0",
+  "description": "React 18 test application and playground",
+  "version": "1.1.0",
```

Comment:
Not sure if this is needed, but since this app changes its behavior, it makes sense to bump.

Reply:
no need to do this, ideally these kind of "app" abuse is lives co-located , but with this new aproach it makes sense. 

we should consider all these apps to have `0.0.0`

**action:** please undo the version change